### ISSUE TITLE
[Bug] Apostrophe character (U+2019 ’ Right Single Quotation Mark) in names does not display correctly

### ISSUE THREAD

**Issue by jroyvde** [zyrouge/symphony#706]
### Description

A variant of the apostrophe character (`’` as opposed to `'`), which seems to be commonly used in album & song titles, does not display correctly in the app. This occurs regardless of which font is selected in the app's settings. The issue may also apply to other less-common unicode characters, but I haven't done any further testing.

### Steps to Reproduce

Add a song with the `’` character in its name to Symphony.

### Expected Behavior

The character should display normally.

### Screenshots

<img src="https://github.com/user-attachments/assets/a734a6b5-bd29-4a4e-9ced-4302b5aa7062" width=25% height=25%>
</br>

<img src="https://github.com/user-attachments/assets/44510d8e-3475-4bbf-8b73-078436527eb9" width=50% height=50%>

### Device

Google Pixel 6a

### OS

Android 15

### Version

2024.11.114

### Contribution Guidelines

- [X] I agree to follow the [Contribution Guidelines](https://github.com/zyrouge/symphony/wiki/Contributions-Guidelines#issues).

**Comment by jroyvde** [comment]
The song's ID is `1733976723231.0.mr`, don't know if the .mr part is the suffix you mentioned or not.

I've checked it out in a metadata viewer and it looks OK there.

![image](https://github.com/user-attachments/assets/beb7d600-bae5-4e98-8510-5214e4a25448)

**Comment by zyrouge** [comment]
> The song's ID is `1733976723231.0.mr`, don't know if the .mr part is the suffix you mentioned or not.
> 
> I've checked it out in a metadata viewer and it looks OK there.
> 
> ![image](https://private-user-images.githubusercontent.com/54386705/395007302-beb7d600-bae5-4e98-8510-5214e4a25448.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzM5NzgwMTksIm5iZiI6MTczMzk3NzcxOSwicGF0aCI6Ii81NDM4NjcwNS8zOTUwMDczMDItYmViN2Q2MDAtYmFlNS00ZTk4LTg1MTAtNTIxNGU0YTI1NDQ4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjEyVDA0MjgzOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY5NDRlYzAyMDNhNTgxN2ZlNjYyNzE3NWMzMjYwMzgzYmYyMWZhMDU4YjM2NWMwN2Q2OWVhMTk5MTdhY2FjNTcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.JyefFoKLPgpEwRj6HlHpnlZn8dHWW8LigMCWwnZteKA)

Yup, that suffix is what I was asking for. Seems like Metaphony could not parse that song. Could you send the song here in a zip?

**Comment by jroyvde** [comment]
Sure, here it is.

[08. Cynthia Erivo - I’m Not That Girl.zip](https://github.com/user-attachments/files/18106735/08.Cynthia.Erivo.-.I.m.Not.That.Girl.zip)

**Comment by gabeweb** [comment]
Hi,

I had a similar issue to this (but with other characters).

The artist is called "+/-" ([Plus/Minus](https://www.last.fm/music/%252B%2F-)) but Symphony just recognize it as "-", and deactivating the Metaphony metadata decoder, when I played that artist, crashed it.

![Screenshot_2025-01-15-17-52-13-127_io github zyrouge symphony](https://github.com/user-attachments/assets/81449820-2960-49ed-a23b-f2484aed03b3)

Version:

Symphony v2024.12.115

Tested on:

(Xiaomi) Redmi Note 13 (Android 15/HyperOS 2)
